### ISSUE TITLE
Update Cypress._ to use lodash instead of underscore

### ISSUE THREAD

**Issue by jennifer-shehane** [cypress-io/cypress#548]
### Current behavior:
Currently, Cypress automatically includes [Underscore](http://underscorejs.org/) and exposes it as `Cypress._.` where you can call any valid Underscore method on `Cypress._`.

### New behavior:
Cypress has updated our internal libraries including the driver to use [lodash](https://lodash.com/). It then seemed unnecessary to include both Underscore and lodash in order to maintain `Cypress._` current use. 

Also, since any library can be imported, the use of `Cypress._` is not as essential as it once was.

If you want to continue using Underscore methods in your tests, you can include the following after `npm install --save-dev underscore`:

```javascript
var _ = require('underscore')

it('calls an underscore method', function(){
  cy.request('https://jsonplaceholder.typicode.com/users').then(function(response){
    var ids = _.chain(response.body).pluck('id').first(3).value()
    expect(ids).to.deep.eq([1, 2, 3])
  })
})
```

**Comment by RandallKent** [comment]
The code for this is done, but this has yet to be released. We'll update the issue and reference the changelog when it's released.

**Comment by brian-mann** [comment]
Fixed in [`0.20.0`](https://docs.cypress.io/guides/references/changelog.html#0-20-0)
